### DISCUSSION TITLE
point_0688

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#220] [Roy]
"20200522T071619_20200522T071820_T41ULB"
![image](https://user-images.githubusercontent.com/54947195/147492354-3d3acca2-092b-41a6-a466-9b4e52a26694.png)
![image](https://user-images.githubusercontent.com/54947195/147492449-7a004292-677f-4134-8341-ac4754d6a719.png)

"20200404T070621_20200404T070943_T41ULB"
![image](https://user-images.githubusercontent.com/54947195/147492948-f45b65aa-91a2-4b67-8098-6d0dd085c47a.png)

"20190719T070631_20190719T071006_T41ULB"
![image](https://user-images.githubusercontent.com/54947195/147493150-f25dd88a-5507-466b-9d11-f12b0328849a.png)
![image](https://user-images.githubusercontent.com/54947195/147493262-4e181271-d381-44b1-9b1e-f5a5763cff38.png)

"20181216T071309_20181216T071307_T41ULB"
![image](https://user-images.githubusercontent.com/54947195/147493407-45f00d16-9d04-4f39-b6a7-bceefe4b4fd5.png)
![image](https://user-images.githubusercontent.com/54947195/147493456-becaf200-decd-4e62-9b16-37b3c116383d.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20200404T070621_20200404T070943_T41ULB"

![123](https://user-images.githubusercontent.com/86502862/152458997-67105251-5315-41c4-bcf7-5c9c9a55aefa.png)
